reference/constraints/Traverse.rst

```diff
@@ -0,0 +1,98 @@
+Traverse
+========
+
+Objects do not validate nested objects by default unless explicitly using
+this constraint.
+If only specific nested objects should be validated by cascade, consider
+using the :doc:`references/constraints/Valid` instead.
+
++----------------+-------------------------------------------------------------------------------------+
+| Applies to     | :ref:`class <validation-class-target>`                                              |
++----------------+-------------------------------------------------------------------------------------+
+| Options        | - `payload`_                                                                        |
++----------------+-------------------------------------------------------------------------------------+
+| Class          | :class:`Symfony\\Bridge\\Doctrine\\Validator\\Constraints\\Traverse`                |
```

Comment:
Is the namespace valid? I find it here: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Validator/Constraints/Traverse.php 🤔

Reply:
Also, why isn't there any dedicated validator for this specific constraint?

Reply:
See #11320. Like `Valid` those constraints are handled by the recursive context itself to define the cascade strategy, so do not need custom validator.

Reply:
Ooh, TIL, thanks 😄